src/libraries/System.IO.Compression.ZipFile/src/System.IO.Compression.ZipFile.csproj

```diff
@@ -11,7 +11,7 @@
     <Compile Include="System\IO\Compression\ZipFileExtensions.ZipArchive.Create.cs" />
     <Compile Include="System\IO\Compression\ZipFileExtensions.ZipArchive.Extract.cs" />
     <Compile Include="System\IO\Compression\ZipFileExtensions.ZipArchiveEntry.Extract.cs" />
-    <Compile Include="$(CommonPath)System\IO\PathInternal.CaseSensitivity.cs"
+    <Compile Include="$(CoreLibSharedDir)System\IO\PathInternal.CaseSensitivity.cs"
```

Comment:
Would `"$(CommonPath)System\IO\PathInternal.*` be more appropriate place for these files? 

These files do not contain public APIs. I think we only use CoreLib includes for files with public APIs, and only include them for legacy TFM configurations.